database/migrations/2023_02_08_193056_products.php

```diff
@@ -12,7 +12,23 @@
      */
     public function up()
     {
-
+        Schema::create('products', function (Blueprint $table) {
+            $table->id();
+            $table->string('name')->unique();
```

Comment:
remove name and slug columns from every table 
because we get them from localizations table